src/Tasks/Microsoft.NET.Build.Tasks/GenerateRuntimeConfigurationFiles.cs

```diff
@@ -179,7 +177,7 @@ protected override void ExecuteCore()
                                    LockFileTargetLibrary lockFilePlatformLibrary,
                                    bool isFrameworkDependent)
         {
-            runtimeOptions.Tfm = TargetFramework;
+            runtimeOptions.Tfm = NuGetFramework.Parse(TargetFrameworkMoniker).GetShortFolderName();
```

Comment:
Removing this parameter will fail when the .targets try to pass it to the task: https://github.com/dotnet/sdk/blob/ee31a03ce4b6d58d6fca1505eced1f0f5e73bfd9/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets#L292